app/interactors/send_team_hours_update.rb

```diff
@@ -0,0 +1,15 @@
+class SendTeamHoursUpdate
+  include Interactor
+  include HasHarvest
+
+  def call
+    Team.active.each do |team|
+      from = Date.this_week.first
+      to = Date.this_week.last
```

Comment:
Contemplated doing that but didn't as that hasn't been the pattern so far. I could make it possible with defaults of `this_week`?